systemservices/marketplace/src/types/version.ts

```diff
@@ -2,8 +2,6 @@ import { Manifest } from "./manifest";
 
 export interface Version {
   versionHash: string;
-  manifest: string;
-  manifestProtocol: string;
-  manifestData: Manifest|undefined;
+  manifest: Manifest;
```

Comment:
I'm not sure removing basic data like `manifestProtocol` and `manifest` is a good thing.
I see the advantage of creating a separate layer between the api of the smart contract and the api of this service, but here it could still be useful to return the manifest source "url".

Reply:
I don't see any reasons to return these information if it is information managed by the service itself. The question to ask is will we ever need this information to access the data if we always have the data.

Reply:
> Only returning the manifest's data but not manifestProtocol and manifestData is too much for my point of view. Those 2 info, even if they are completely managed by the service, are still valid to return.
As the marketplace is on Ethereum, anyone can add a new version with wrong manifest data. In this PR's current implementation, the version is discarded and it will appear that the service has no version, even if it actually has one. The wrong version cannot be deployed on a core, but should still be return as a version with wrong manifest. Othercase: the manifest is valid but for some reason is not available on IPFS anymore?
I think returning the version even if the manifest is not valid / available is better than just discard it.

Even more, what about service that have the manifest in private host that require authentication?